cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/transformers/AbstractCgmesTapChangerBuilder.java

```diff
@@ -47,28 +46,21 @@ static CgmesPhaseTapChangerBuilder newPhaseTapChanger(PropertyBag phaseTapChange
         return new CgmesPhaseTapChangerBuilder(phaseTapChanger, xtx, context);
     }
 
-    protected int initialTapPosition(int defaultStep) {
-        switch (context.config().getProfileForInitialValuesShuntSectionsTapPositions()) {
```

Comment:
I don't know if the parameter `PROFILE_FOR_INITIAL_VALUES_SHUNT_SECTIONS_TAP_POSITIONS` should not be deleted now ?